.travis.yml

```diff
@@ -16,29 +16,57 @@ addons:
 
 env:
   global:
-  - DATABASE_URL='postgres://postgres@localhost/usaspending_api'
+  - USASPENDING_DB_USER=postgres
+  - DATABASE_URL=postgres://${USASPENDING_DB_USER}@localhost/usaspending_api
   - DJANGO_SETTINGS_MODULE='usaspending_api.settings'
   - ES_HOSTNAME='http://localhost:9200'
+  - BROKER_DB_HOST=localhost
+  - BROKER_DB_PORT=5435
+  - BROKER_DB_USER=admin
+  - BROKER_DB_PASSWORD=root
+  - BROKER_DB_DBNAME=data_broker
+  - DATA_BROKER_DATABASE_URL=postgres://${BROKER_DB_USER}:${BROKER_DB_PASSWORD}@${BROKER_DB_HOST}:${BROKER_DB_PORT}/${BROKER_DB_DBNAME}
+  - BROKER_REPO_FOLDER=${TRAVIS_BUILD_DIR}/../data-act-broker-backend
 
 before_install:
   - npm install dredd@5.4.1 --global
 
 install:
+  - apt-get install -y gettext-base  # for envsubst
   - if [[ $TRAVIS_PYTHON_VERSION == '3.7' ]]; then travis_retry pip install -r requirements/requirements_3-7.txt; else travis_retry pip install -r requirements/requirements.txt; fi
   - pip install coveralls
+  # Checkout dependent broker code used to spin up a broker integration test db. Put it in its own folder alongside this repo's code
+  - git clone --branch ftr/dev-1526-broker-integ-test-db --single-branch --depth 1 https://github.com/fedspendingtransparency/data-act-broker-backend.git ${BROKER_REPO_FOLDER}
 
 before_script:
   - curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.1.1.deb && sudo dpkg -i --force-confnew elasticsearch-6.1.1.deb && sudo service elasticsearch restart
-  - psql -c "CREATE DATABASE usaspending_api;" -U postgres
+  - sleep 10  # recommended by Travis to ensure Elasticsearch has time to spin up
+  - psql -c "CREATE DATABASE usaspending_api;" -U ${USASPENDING_DB_USER}
   - psql -c "CREATE USER readonly;"
+  # Establish foreign server for dblink to Broker
+  - psql --file usaspending_api/database_scripts/extensions/extensions.sql
+  - |
+    eval "cat <<EOF
+    $(<usaspending_api/database_scripts/broker_server.sql)
+    EOF
+    " > usaspending_api/database_scripts/broker_server.sql
+  - psql --file usaspending_api/database_scripts/broker_server.sql
```

Comment:
Missing "servers" in path: usaspending_api/database_scripts/servers/broker_server.sql

Could the SQL-substituted string be piped into psql instead of into the sql file?